api/api.go

```diff
@@ -40,14 +40,15 @@ func New(
 	logDB *logdb.LogDB,
 	bft bft.Finalizer,
 	nw node.Network,
+	forkConfig thor.ForkConfig,
 	allowedOrigins string,
 	backtraceLimit uint32,
 	callGasLimit uint64,
 	pprofOn bool,
 	skipLogs bool,
 	allowCustomTracer bool,
-	isReqLoggerEnabled bool,
-	forkConfig thor.ForkConfig,
+	enableReqLogger bool,
+	enableMetrics bool,
```

Comment:
`isReqLoggerEnabled` is a boolean bc it's a question. But I understand not everyone's' cup of tea.

Perhaps the past tense ? `enabledReqLogger` and `enabledMetrics`?

Reply:
For past tense, `reqLoggerEnabled` and `metricsEnabled`, the current one just followed the flag's name convention.

Reply:
Yeah that's true, sounds good 👍